File: Cine/gestorAplicación/cine/Pelicula.java
package gestorAplicación.cine;

import java.io.*;

public class Pelicula implements Serializable {
	
	private static final long serialVersionUID = 1L;
	
	private String nombre;
	
	private String genero;
	

	public Pelicula(String nombre, String genero) {
		
		this.nombre = nombre;
		
		this.genero = genero;
		
	}
	
	public String getnombre() {
		
		return nombre;		
		
	}
	    
	public String getGenero() {
		
		return genero;		
		
	}

}
	

File: Python/src/gestionAplicacion/cinepy/Pelicula.py
class Pelicula:

    pelicula = []

    def __init__(self, nombre, genero):
        self._nombre = nombre
        self._genero = genero

    def getNombre(self):
        return self._nombre
    
    def getGenero(self):        
        return self._genero
    
    def setNombre(self, nombre):
        self._nombre = nombre

    def setGenero(self, genero):
        self._genero = genero

    @staticmethod
    def getPelicula():
        return Pelicula.pelicula
    
    @staticmethod
    def setPelicula(pelicula):
        Pelicula.pelicula.append(pelicula)
    
        